core/storage/statistics/gae_models.py

```diff
@@ -64,119 +59,6 @@ def get_or_create(cls, exploration_id, state_name):
         return counter
 
 
-class StateRuleAnswerLogModel(base_models.BaseModel):
```

Comment:
Discussed over Hangouts with you: I tested deploying this to the migration server and everything seemed to work fine, including deleting entities from the removed models through datastore admin. We should be okay with just removing old models, though actually going into datastore admin and cleaning up the entities might as well be on a best-effort basis.